linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -133,7 +136,7 @@ public static void main(String[] arguments)
       } catch (ModelBuildingException ex) {
         // When there's a pom.xml file that is not (indirectly) referenced by the root pom.xml,
         // Maven may fail to build the model. Such pom.xml can be ignored.
-        System.out.println("Ignoring bad model: " + path);
+        logger.warning("Ignoring bad model: " + path + ": " + ex.getMessage());
```

Comment:
should this even be a warning? Do we expect the user to act on it? If not, we don't need to notify them.

Reply:
Lowered to info. This information of un-build-able pom.xml is useful when troubleshooting.